### ISSUE TITLE
cannot get pypy and pypy3 to build on os x

### ISSUE THREAD

**Issue by mmerickel** [collective/buildout.python#38]
I'm on mavericks and running `bin/buildout -c local.cfg -v` and I see the appropriate platform detections: `Detected these platforms: darwin-mavericks, darwin, default`. The following is my cfg.

```ini
[buildout]
extends =
    src/base.cfg
    src/pypy.cfg
    src/pypy3.cfg

parts =
    ${buildout:base-parts}
    ${buildout:pypy-parts}
    ${buildout:pypy3-parts}

python-buildout-root = ${buildout:directory}/src
```

The same style works great for CPython. I also got it to work by adding `parts = pypy` directly instead of depending on the os-specific `pypy:darwin-mavericks` part that is not being properly included.

I'm not familiar with how the osdetect integrates with buildout but something is preventing it from working properly for me. At least the `[buildout:darwin]` section doesn't appear to be getting included.

**Comment by jean** [comment]
I'm believe I'm seeing the same, buildout stops with:

```
[...]
Finished processing dependencies for Pillow
Updating python-3.5-test.
Installing pypy-download.
While:
  Installing pypy-download.
Error: Missing option: pypy-download:url
```

> System:    Host: UX303LB Kernel: 3.19.0-18-generic x86_64 (64 bit) Desktop: MATE 1.10.2  Distro: Linux Mint 17.2 Rafaela

**Comment by mauritsvanrees** [comment]
I have hopefully committed a fix on master, commit 9e4ddcbc573a1728eddffa28aee6b21af00c5814.
Can you check if that helps?

**Comment by mauritsvanrees** [comment]
As said, this should work now, so I am closing this.
Please reopen if you still have problems.
